mantle/cmd/kola/options.go

```diff
@@ -117,14 +117,14 @@ func init() {
 	sv(&kola.GCEOptions.JSONKeyFile, "gce-json-key", "", "use a service account's JSON key for authentication")
 
 	// openstack-specific options
-	sv(&kola.OpenStackOptions.ConfigPath, "openstack-config-file", "", "OpenStack config file (default \"~/"+auth.OpenStackConfigPath+"\")")
-	sv(&kola.OpenStackOptions.Profile, "openstack-profile", "", "OpenStack profile (default \"default\")")
+	sv(&kola.OpenStackOptions.ConfigPath, "openstack-config-file", "", "Path to a clouds.yaml formatted OpenStack config file. The underlying library defaults to ./clouds.yaml")
+	sv(&kola.OpenStackOptions.Profile, "openstack-profile", "", "OpenStack profile within clouds.yaml (default \"openstack\")")
 	sv(&kola.OpenStackOptions.Region, "openstack-region", "", "OpenStack region")
 	sv(&kola.OpenStackOptions.Image, "openstack-image", "", "OpenStack image ref")
 	sv(&kola.OpenStackOptions.Flavor, "openstack-flavor", "1", "OpenStack flavor ref")
 	sv(&kola.OpenStackOptions.Network, "openstack-network", "", "OpenStack network")
 	sv(&kola.OpenStackOptions.Domain, "openstack-domain", "", "OpenStack domain ID")
-	sv(&kola.OpenStackOptions.FloatingIPPool, "openstack-floating-ip-pool", "", "OpenStack floating IP pool for Compute v2 networking")
+	sv(&kola.OpenStackOptions.FloatingIPNetwork, "openstack-floating-ip-network", "", "OpenStack network to use when creating a floating IP")
```

Comment:
Do we need to hold backwards compatibility with `openstack-floating-ip-pool`? IE if `openstack-floating-ip-pool` is sued, translate it (if that's possible) and warn about deprecation? Or is this a breaking change we must do to move forward?

Reply:
I don't know of any users of this other than us and we haven't been using this openstack code for a while. I think we should just drop the dead weight and move on. I'd be shocked if anyone notices us dropping `--openstack-floating-ip-pool`.

Reply:
SGTM